plugins/tedge_apt_plugin/tests/main.rs

```diff
@@ -72,6 +72,7 @@ fn run_cmd(cmd: &str, args: &str) -> anyhow::Result<(String, String, i32)> {
 // expected exit code
 //
 // description about the test
+#[ignore] // ignored, requires sudo
```

Comment:
How about disabling these tests with a feature flag like `sudo-test` instead of this hard `ignore` so that they can still be run locally if someone wants to, by explicitly providing this flag?